apps/picsa-apps/dashboard/src/app/modules/translations/pages/edit/translations-edit.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="page-content">
+  <h2>Edit Translations</h2>
+  @if(translationRow){
+  <form class="form-content">
+    <div class="form-data">
+      <label>Date of Creation</label>
+      <div>{{ translationRow.created_at | date: 'mediumDate' }}</div>
```

Comment:
(+) - Nice use of the date pipe to display a friendlier format, I had forgotten all about that one!